subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -11,18 +11,21 @@ config BT_LLL_VENDOR_NORDIC
 	depends on !$(dt_nodelabel_enabled,timer0)
 	depends on !$(dt_nodelabel_enabled,rtc0)
 
-	select ENTROPY_NRF5_RNG
-	select ENTROPY_NRF5_BIAS_CORRECTION
+	select ENTROPY_NRF5_RNG if BT_CTLR_CRYPTO
+	select ENTROPY_NRF5_BIAS_CORRECTION if ENTROPY_NRF5_RNG
 
 	select BT_HAS_HCI_VS
-	select BT_CTLR_LE_ENC_SUPPORT if !BT_CTLR_DATA_LENGTH_CLEAR && \
+	select BT_CTLR_CRYPTO_SUPPORT if !SOC_SERIES_NRF54LX
```

Comment:
@wopu-ot or @thoh-ot  This PR introduces explicit need to select `BT_CTLR_CRYPTO_SUPPORT` in out-of-tree LLL variants.